src/pages/createPost.tsx

```diff
@@ -154,122 +101,166 @@ const createPost = () => {
+                        <InputAdornment position="start">
+                          <CalendarTodayIcon />
+                        </InputAdornment>
+                      ),
+                    }}
+                  />
+                </LocalizationProvider>
+                <div style={helperTextBox}>
+                  <FormHelperText sx={helperTextError}>
+                    ช่องนี้ไม่สามารถเว้นว่างได้
+                  </FormHelperText>
+                </div>
+              </Grid>
+            </Grid>
           </Grid>
-
-
           <Grid item>
             <Typography variant="h5">Tag</Typography>
             <Autocomplete
-            multiple
-            limitTags={5}
-            id="multiple-limit-tags"
-            options={Tags}
-            getOptionLabel={(option) => option.Tag}
-            renderInput={(params) => (
-              <TextField {...params} placeholder="Click for add tag" fullWidth />
-            )}
-            
+              multiple
+              options={Tags}
+              value={selectedTags}
+              onChange={handleAddTag}
+              renderInput={(params) => (
+                <TextField
+                  {...params}
+                  placeholder="Click for add tag"
+                  fullWidth
+                />
+              )}
             />
-            
-            
           </Grid>
-          
-
-
-
           <Grid item>
-          <Typography variant="h5">Description</Typography>
-          <TextField variant="outlined" fullWidth size="Big" placeholder="เช่น มาเที่ยวกันเลย ร้านบอร์ดเกมแถวรัชดา" />
+            <Typography variant="h5">Description</Typography>
+            <TextField
+              variant="outlined"
+              fullWidth
+              multiline
+              rows={4}
+              size="medium"
+              placeholder="เช่น มาเที่ยวกันเลย ร้านบอร์ดเกมแถวรัชดา"
+            />
             <div style={helperTextBox}>
-              <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
+              <FormHelperText sx={helperTextError}>
+                ช่องนี้ไม่สามารถเว้นว่างได้
+              </FormHelperText>
               <FormHelperText sx={helperText}>0/500</FormHelperText>
             </div>
-        </Grid>
-        <Grid item>
-          <Typography variant="h5">Image(Optional)</Typography> 
-          
-        </Grid>
-
-
+          </Grid>
+          <Grid item>
+            <Typography variant="h5">Image(Optional)</Typography>
+            <Stack
+              direction="row"
+              justifyContent="flex-start"
+              alignItems="flex-start"
+              spacing={1}
+            >
+              <PictureList />
+            </Stack>
+          </Grid>
         </div>
         <Grid item>
-          <Button variant="contained" sx={saveBtn}>
-            <Typography variant="subtitle1">Create Post</Typography>
+          <Button variant="contained" sx={{ color: "primary" }}>
```

Comment:
If you have read some MUI's Button API docs, you should find out that using "color" attribute is cleaner than "sx". Please read docs before using each component.